docs/platform/concepts/tax-information.rst

```diff
@@ -0,0 +1,46 @@
+Tax information regarding Aiven services
+========================================
+
+Aiven services are provided by Aiven Ltd, a private limited company incorporated in Finland. 
+This article describes our tax status in different jurisdictions. 
+None of our marketed prices include value-added or other taxes.
+
+European Union
+--------------
+
+The Finnish law requires us to charge a value-added tax for services provided within the European Union. 
+The value-added tax percentage depends on the domicile of the customer.
+
+For business customers in EU countries, other than Finland, we can apply the reverse charge mechanism of 2006/112/EC article 196, 
+when a valid VAT ID is entered into the billing information of the relevant Aiven project.
+
+Rest of the world
+-----------------
+
+No value-added tax is applied to services sold outside the European Union as described in Value-Added Tax Act of Finland section 69h. 
+Finland has tax treaties with most countries in the world affirming this status.
```

Comment:
Finland has tax treaties with most countries in the world, affirming this status. (comma added)